doc/api/net.md

```diff
@@ -586,259 +772,274 @@ the socket is destroyed (for example, if the client disconnected).
 added: v0.11.14
 -->
 
-The string representation of the remote IP family. `'IPv4'` or `'IPv6'`.
+Provides the string representation of the remote IP family. Will be either
+`'IPv4'` or `'IPv6'`.
 
 ### socket.remotePort
 <!-- YAML
 added: v0.5.10
 -->
 
-The numeric representation of the remote port. For example,
-`80` or `21`.
+Provides the numeric representation of the remote port. For example, `80` or
+`21`.
 
 ### socket.resume()
 
-Resumes reading after a call to [`pause()`][].
+The `socket.resume()` method resumes reading after a call to
+[`socket.pause()`][]. See the [Readable][] documentation for more information.
 
 ### socket.setEncoding([encoding])
 <!-- YAML
 added: v0.1.90
 -->
 
-Set the encoding for the socket as a [Readable Stream][]. See
-[`stream.setEncoding()`][] for more information.
+The `socket.setEncoding()` method sets the encoding for the socket as a
+[Readable Stream][]. See [`readable.setEncoding()`][] for more information.
 
 ### socket.setKeepAlive([enable][, initialDelay])
 <!-- YAML
 added: v0.1.92
 -->
 
-Enable/disable keep-alive functionality, and optionally set the initial
-delay before the first keepalive probe is sent on an idle socket.
-`enable` defaults to `false`.
+* `enable` {boolean} `true` to enable keep-alive, `false` to disable. Defaults
```

Comment:
should make it clear these are TCP level keep alives, maybe even explicitly mention SOL_KEEPALIVE so people can google for it, and perhaps warn that it should not be used unless the limitations of the TCP keep alive are understood